build.gradle

```diff
@@ -1148,8 +1155,10 @@ project(':clients') {
 
   task createVersionFile(dependsOn: determineCommitId) {
     ext.receiptFile = file("$buildDir/kafka/$buildVersionFileName")
+    inputs.property('commitId', commitId)
+    inputs.property('version', version)
     outputs.file receiptFile
-    outputs.upToDateWhen { false }
+    outputs.cacheIf { true }
```

Comment:
Since this file is rather small and easy to create. It might make sense to _not_ enable the build cache for this task since the ziping and unzipping of this cache entry might take longer than it would take to write this file directly.

Reply:
Yeah, that's true. In this case, the cost to zip up this small file is also tiny, but either way, there's really no benefit to caching on these tasks since they run so fast anyway.  I'll go ahead and remove this.